packages/astro/src/assets/consts.ts

```diff
@@ -0,0 +1,14 @@
+export const VIRTUAL_MODULE_ID = 'astro:assets';
+export const VIRTUAL_SERVICE_ID = 'virtual:image-service';
+export const VALID_INPUT_FORMATS = [
+	'heic',
+	'heif',
+	'avif',
+	'jpeg',
+	'jpg',
+	'png',
+	'tiff',
+	'webp',
+	'gif',
```

Comment:
nit: I wonder if there's a way to keep these formats in sync with `client-image.d.ts`

Reply:
Apart from type gen, I unfortunately don't think so :( Module declarations are unfortunately super rigid in TypeScript